packages/core/__tests__/src/models.test.ts

```diff
@@ -86,4 +86,21 @@ describe("Manifest", () => {
     );
     expect(decoded).toBeNull();
   });
+
+  test("apiEnpoint: https://foo-bar.cdn.prismic.io/api/v2", () => {
+    const input = { apiEndpoint: "https://foo-bar.cdn.prismic.io/api/v2" };
+    const result = Manifest.decode(input);
+    expect(isRight(result)).toBeTruthy();
+    const decoded = pipe(
```

Comment:
If the test before says "isRight" = true, I think you can do result.right.
Like the either in scala

Reply:
That's not how io-ts works,

it returns an object with a _tag prop that's either "right" or "left"

Reply:
https://github.com/gcanti/fp-ts/blob/master/src/internal.ts#L33